metabase/metabase
# Deduplicate babel dependencies in yarn.lock

### Description

result of running 

```
npx yarn-deduplicate --scopes @babel
```

actually resolves multiple versions of the babel packages to the most recent 

### How to verify 

First:

```
rm -rf node_modules && yarn
```

- tests should pass
- local run should work 
- build should work
- storybook build should work
- release build should work

### Side effects

#### Disk space taken
together with https://github.com/metabase/metabase/pull/35659 cuts ~1700MB of node_modules 

before
<img width="520" alt="image" src="https://github.com/metabase/metabase/assets/125459446/aba281ca-7cc9-40e8-af42-04dd909fbb30">

after
<img width="654" alt="image" src="https://github.com/metabase/metabase/assets/125459446/17d48d5d-1031-457f-a8e8-e683b10c3fab">

#### Time to install node_modules

Before
<img width="257" alt="Screenshot 2023-11-17 at 11 05 37" src="https://github.com/metabase/metabase/assets/125459446/f7ba2667-4a3a-4df7-8135-c6fe238a8f61">

After
<img width="321" alt="Screenshot 2023-11-17 at 11 05 26" src="https://github.com/metabase/metabase/assets/125459446/c76a0fc2-5685-4495-83b0-f0c6d7ee1b08">
